pkg/config/datafileprojectconfig/config.go

```diff
@@ -247,19 +265,31 @@ func NewDatafileProjectConfig(jsonDatafile []byte, logger logging.OptimizelyLogP
 		return nil, err
 	}
 
+	var hostForODP, publicKeyForODP string
+	if len(datafile.Integrations) > 0 {
+		integration := datafile.Integrations[0]
+		if integration.Key == "odp" {
```

Comment:
This seems weird, can we 'iterate` `integrations` and check `odp`, assign it and get out of `loop`